### PR TITLE
Disp color mix

### PR DESCRIPTION
**Pull request by e2002e**
For #2618, without the changes for #2616.

### CODE REVIEW

File: blender/arm/material/make_tess.py [hunk 1]
```diff
@@ -24,6 +24,7 @@ def interpolate(tese, var, size, normalize=False, declare_out=False):
 
     if normalize:
         s += '{0}{1} = normalize({1}_0 + {1}_1 + {1}_2);\n'.format(prep, var)
+        s += 'vec3 n = normalize({0});\n'.format(var)
```
Comment (luboslenco) [inline blender/arm/material/make_tess.py:27]: Do we need normalize() here?

Comment (e2002e) [reply blender/arm/material/make_tess.py:27]: Well no, but in this case n is useless and should be replaced with wposition.
`vec3 TextureCoordinate_Normal_res = n;`
I'm looking into that.

Comment (MoritzBrueckner) [reply blender/arm/material/make_tess.py:27]: The value of `TextureCoordinate_Normal_res` is probably generated in [`nodes_input.parse_texcoord()`](https://github.com/armory3d/armory/blob/34bf363e5f8cfeccde760eefe44d010a2b9c5a4c/blender/arm/material/cycles_nodes/nodes_input.py#L293-L294) (called from within [`cycles.parse_vector()`](https://github.com/armory3d/armory/blob/34bf363e5f8cfeccde760eefe44d010a2b9c5a4c/blender/arm/material/cycles.py#L305)), and the name is generated [here](https://github.com/armory3d/armory/blob/34bf363e5f8cfeccde760eefe44d010a2b9c5a4c/blender/arm/material/cycles.py#L534).

File: blender/arm/material/cycles_nodes/nodes_input.py [hunk 2]
```diff
@@ -103,7 +104,7 @@ def parse_attribute(node: bpy.types.ShaderNodeAttribute, out_socket: bpy.types.N
     return c.cast_value('0.0', from_type='float', to_type=out_type)
 
 
-def parse_rgb(node: bpy.types.ShaderNodeRGB, out_socket: bpy.types.NodeSocket, state: ParserState) -> vec3str:
+def parse_rgb(bnode: bpy.types.ShaderNodeRGB, out_socket: bpy.types.NodeSocket, state: ParserState) -> vec3str:
```
Comment (MoritzBrueckner) [inline blender/arm/material/cycles_nodes/nodes_input.py:107]: Looks like a typo here, in the function you still use `node` everywhere

Comment (e2002e) [reply blender/arm/material/cycles_nodes/nodes_input.py]: fixed, don't know how this still worked

**Pull request merged**